### PR TITLE
Feature tests tweaks

### PR DESCRIPTION
**Pull request by gnott**
Minor changes otherwise the tests fail in Windows.

### CODE REVIEW

File: tests/activity/test_activity_apply_version_number.py [hunk 1]
```diff
@@ -76,8 +76,10 @@ def test_rewrite_xml_file(self, mock_path_join):
         #then
         result_file = open(u'tests/files_dest_ApplyVersionNumber/elife-15224-v1.xml', 'r')
```
Comment (giorgiosironi) [inline tests/activity/test_activity_apply_version_number.py]: what about the idiom `with open(u'tests/files_dest_ApplyVersionNumber/elife-15224-v1.xml', 'r') as result_file:`?

**Comment by Jenniferstrej** [comment]
@gnott , I forgot about your situation with Windows and file open mode, sorry. 
Got point I also forgot to close the file, my bad. Thanks for spotting 👍

**Comment by gnott** [comment]
The "with open(..." works ok too without the separate close() line, to note. Either or - I can change it if there's a preference.

**Comment by giorgiosironi** [comment]
Yes, `with open()` makes it impossible to forgot the `close()` (and in case of exceptions coming up catches them, closes the file, and reraises)

**Pull request merged**